packages/fastify/src/__snapshots__/clerkPlugin.test.ts.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`clerkPlugin() polyfills reply with server response methods 1`] = `
```

Comment:
This snapshot is testing the mock (jest) function - was this your intention?

Reply:
my intention was to verify that the `setHeader` , `writeHead` and `end` was decorated on `reply` .
I could refactor the test to to be :
```javascript
expect(fastify.decorateReply.calls).toMatchSnapshot();
```
so that the expected snapshot would be like this:
```
 [
      "setHeader",
      [Function],
    ],
    [
      "writeHead",
      [Function],
    ],
    [
      "end",
      [Function],
    ]
```
Does this make any sense to you?

Reply:
Since we have removed the polyfill, i will remove this snapshot and the test.